sonic_data_client/mixed_db_client.go

```diff
@@ -75,6 +75,87 @@ type MixedDbClient struct {
 
 var mixedDbClientMap = map[string]MixedDbClient{}
 
+func hget(table swsscommon.Table, key string, field string) (string, error) {
+	var fieldValuePairs = swsscommon.NewFieldValuePairs()
+	var result = table.Get(key, fieldValuePairs)
+    if result {
```

Comment:
Please check all the tab and space in your code.

Reply:
Fixed